### DISCUSSION TITLE
LoRA Block Weight

### DISCUSSION THREAD

**Discussion by wpsfuyun** [Comfy-Org/ComfyUI#677] [Q&A]
May I ask if it is possible to perform hierarchical control of Lora in comfyui

https://github.com/hako-mikan/sd-webui-lora-block-weight

**Comment by ltdrdata** [comment]
I submit PR that implements the LoRA blockweight, but it is currently on hold due to considerations for structural changes such as SDXL.

https://github.com/comfyanonymous/ComfyUI/pull/303

**Reply by wpsfuyun** [reply]
> 我提交了实现LoRA blockweight的PR，但是目前由于考虑到SDXL等结构变化而被搁置。
> 
> #303

Okay, thank you for your efforts

**Reply by moon47usaco** [reply]
One more vote for LoRA block weight...

**Comment by ltdrdata** [comment]
> One more vote for LoRA block weight...

I've implemented a draft of the lora block weight here. Please give it a try and provide feedback. This is a test node, so after improving its usability, I'll integrate it into the impact pack.

https://github.com/ltdrdata/ComfyUI-Impact-Exp

**Reply by xxlukexx** [reply]
Thank you for this, I'm trying it out now. 

Would you like feedback here in this thread, or on the repo page?

**Reply by ltdrdata** [reply]
> Thank you for this, I'm trying it out now. 
> 
> Would you like feedback here in this thread, or on the repo page?

I think repo page is more appropriate. :)

**Reply by moon47usaco** [reply]
This is excellent. Just a quick test on a random model and lora. Works without error and all the presets work as expected... =]
